finality-aleph/Cargo.toml

```diff
@@ -10,6 +10,8 @@ aleph-bft = "0.18.1"
 aleph-bft-rmc = "0.4.0"
 aleph-primitives = { package = "primitives", path = "../primitives" }
 
+aleph-aggregator = { path = "../aggregator", package = "aggregator" }
```

Comment:
Hmm, if we want this to work as advertised `aleph-aggregator` should get published to crates.io and we have to depend on a given version, not pathwise. I'm pretty sure you can add an explicit version in this specification here already, even if the rest of the infrastructure is not yet done.

Also, figure out how to automatically publish? Probably as a separate task.

Reply:
I was thinking of just using commit/tag at first then you don't need to publish it yet to crate.io and can select version from github repository

Reply:
Makes some sense, although crates.io would likely be preferable.

Also, even then you probably want to start immediately and include the tag here.

Reply:
I would mark crates.io as a necessary/desired improvement. I will do the tag version first, after this pr is merged because pushing tag to GitHub with not yet finalized version of the pr will be a mess imo